src/server/models/okta/Token.ts

```diff
@@ -0,0 +1,10 @@
+import { User } from '@/server/models/okta/User';
+
+export interface Token {
+  stateToken?: string;
+  sessionToken?: string;
+  expiresAt: string;
+  _embedded?: {
```

Comment:
wondering if a small doc comment on this or link to the docs might be useful to explain what _embedded is

Reply:
have linked do the docs in a comment